uber/models.py

```diff
@@ -876,6 +965,48 @@ def insert_test_admin_account(self):
 
             return True
 
+        def bulk_insert(self, models):
```

Comment:
Ooh, this is really cool. We should alter our reports plugin to take advantage of this.